deps/npm/appveyor.yml

```diff
@@ -0,0 +1,36 @@
+environment:
+  matrix:
+    # LTS is our most important target
+    - nodejs_version: "4"
+    # latest
+    - nodejs_version: "5"
+    # I like 0.10 better than 0.12
+    - nodejs_version: "0.10"
+    - nodejs_version: "0.12"
+    # EOL summer 2016, most likely
+    - nodejs_version: "0.8"
+  COVERALLS_REPO_TOKEN:
+    secure: XdC0aySefK0HLh1GNk6aKrzZPbCfPQLyA4mYtFGEp4DrTuZA/iuCUS0LDqFYO8JQ
```

Comment:
@iarna ... I'm not too familiar with coveralls but I want to verify... was including the `COVERALLS_REPO_TOKEN` here and the slack incoming_webhook secure token intentional?